src/main/java/team/gif/robot/subsystems/drivers/RobotCompressor.java

```diff
@@ -0,0 +1,18 @@
+package team.gif.robot.subsystems.drivers;
+
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import team.gif.robot.RobotMap;
+
+public class RobotCompressor extends Compressor {
+    public static AnalogInput pressureSensor = null;
+    public RobotCompressor(int port, PneumaticsModuleType moduleType) {
+        super(port, moduleType);
+        pressureSensor = new AnalogInput(RobotMap.SENSOR_AIR_PRESSURE);
```

Comment:
I know you copied this from previous years, but we should change RobotMap.SENSOR_AIR_PRESSURE to an argument.
If NULL, then don't instantiate the pressure sensor

Reply:
So, the RobotMap.SENSOR_AIR_PRESSURE should be changed to NULL, right?

Reply:
No ...
```suggestion
public RobotCompressor(int port, PneumaticsModuleType moduleType, int sensorID) {
        pressureSensor = new AnalogInput(sensorID);
```

and in Robot.java

compressor = new RobotCompressor(RobotMap.COMPRESSOR, PneumaticsModuleType.CTREPCM, RobotMap.SENSOR_AIR_PRESSURE);